src/handlers/sendEmailRequestToNotify.spec.ts

```diff
@@ -0,0 +1,135 @@
+}
+const givenSuccessfulSendEmailRequest = async () => {
+  mockSendEmail.mockResolvedValue(testSuccessfulNotifyResponse)
+}
+const givenUnsuccessfulSendEmailRequest = async () => {
+  mockSendEmail.mockImplementation(() => {
+    throw new Error('There was an error sending request to Notify')
+  })
+}
+const validEventBody = `{
+      "email": "${TEST_NOTIFY_EMAIL}",
+      "firstName": "${TEST_NOTIFY_NAME}",
+      "zendeskId": "${TICKET_ID}",
+      "signedUrl": "${TEST_SIGNED_URL}"
+    }`
+const callHandlerWithBody = async (customBody: string) => {
+  return await handler({
+    ...defaultApiRequest,
+    body: customBody
+  })
+}
+
+describe('initiate sendEmailRequest handler', () => {
+  beforeEach(() => {
+    jest.spyOn(global.console, 'error')
+  })
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('creates a NotifyClient and calls sendEmail with correct parameters', async () => {
+    jest.spyOn(global.console, 'log')
+    await givenNotifySecretsAvailable()
+    await givenSuccessfulSendEmailRequest()
+
+    await callHandlerWithBody(validEventBody)
+
+    expect(NotifyClient).toHaveBeenCalledWith('myNotifyApiKey')
+    expect(mockSendEmail).toHaveBeenCalledTimes(1)
+    expect(mockSendEmail).toHaveBeenCalledWith(
+      ALL_NOTIFY_SECRETS.notifyTemplateId,
+      TEST_NOTIFY_EMAIL,
+      {
+        personalisation: {
+          firstName: TEST_NOTIFY_NAME,
+          zendeskId: TICKET_ID,
+          signedUrl: TEST_SIGNED_URL
+        }
+      }
+    )
+    expect(console.log).toHaveBeenCalledWith({
+      status: 201,
+      emailSentTo: TEST_NOTIFY_EMAIL,
+      subjectLine: 'Your data query has completed'
+    })
+  })
+  it('returns a 400 status code with a message', async () => {
+    await givenNotifySecretsAvailable()
+    await givenUnsuccessfulSendEmailRequest()
+
+    await callHandlerWithBody(validEventBody)
+
+    expect(NotifyClient).toHaveBeenCalledWith('myNotifyApiKey')
+    expect(mockSendEmail).toThrowError()
+    expect(mockSendEmail).toHaveBeenCalledWith(
+      ALL_NOTIFY_SECRETS.notifyTemplateId,
+      TEST_NOTIFY_EMAIL,
+      {
+        personalisation: {
+          firstName: TEST_NOTIFY_NAME,
+          zendeskId: TICKET_ID,
+          signedUrl: TEST_SIGNED_URL
+        }
+      }
+    )
+    expect(console.error).toHaveBeenCalledWith(
+      'There was an error sending a request to Notify: ',
+      Error('There was an error sending request to Notify')
+    )
+  })
+  it('returns from the function and logs an error when no event body is present', async () => {
+    const invalidEventBody = ''
+
+    await callHandlerWithBody(invalidEventBody)
+
+    expect(console.error).toHaveBeenCalledWith(
+      'Could not find event body. An email has not been sent'
+    )
+    expect(NotifyClient).not.toHaveBeenCalled()
+  })
+  it('returns from the function and logs an error when a value is missing from the event body', async () => {
+    const invalidEventBody = `{
```

Comment:
I know it's a pain, but it would be good to test the case of each property missing, rather than just one.
You can make this very slightly neater by using an array of property names, and using `map` to run the test for each.